src/js/component.js

```diff
@@ -1181,7 +1190,7 @@ class Component {
     };
 
     // Inherit from this object's prototype
-    subObj.prototype = Lib.obj.create(this.prototype);
+    subObj.prototype = Object.create(this.prototype);
```

Comment:
Yep, es5-sham should have it. 
Was wondering whether it was required so that maybe we could limit us to only have es5-shim.
https://github.com/videojs/video.js/pull/2139#issuecomment-103176907